src/inc/eventtracebase.h

```diff
@@ -216,11 +280,21 @@ extern "C" {
 #endif //!DONOT_DEFINE_ETW_CALLBACK && !DACCESS_COMPILE
 
 #include <clretwallmain.h>
+
+#elif defined(PLATFORM_UNIX)
```

Comment:
Platform linux is defined as __LINUX__. Lets change this to reflect the actual OS.